src/com/facebook/buck/swift/SwiftBuckConfig.java

```diff
@@ -49,6 +49,8 @@
   public static final String ADD_XCTEST_IMPORT_PATHS = "add_xctest_import_paths";
   public static final String SKIP_PREBUILD_TARGET_SUFIX = "skip_prebuild_target_sufix";
   public static final String INDEXING_WHILE_BUILDING = "indexing_while_building";
+  public static final String LOCAL_INDEX_STORE = "local_index_store";
```

Comment:
Could you rename this setting to `global_index_store_enabled`? IMHO makes more sense than `local_index_store`.